### PR TITLE
Fix setting initials extra

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

File: vue/components/personalization.vue [hunk 1]
```diff
@@ -207,7 +204,7 @@ export const personalization = {
             this.initialOptions = Object.assign({}, options);
         });
 
-        this.$bus.bind("initials_extra", initialsExtra => {
+        this.$bus.bind("initials_change", initialsExtra => {
```
Comment (joamag) [inline vue/components/personalization.vue]: why the name change? If not properly justified this is a hot spot for regressions. Please elaborate on the semantics of each of the events.

File: vue/components/personalization.vue [hunk 2]
```diff
@@ -149,10 +149,7 @@ export const personalization = {
         state: {
             handler: function(state, previousState) {
                 if (this.diffInitialsExtra(state.initialsExtra, previousState.initialsExtra)) {
-                    this.$bus.trigger(
-                        "initials_change",
-                        this.sanitizeInitialsExtra(state.initialsExtra)
-                    );
+                    this.$bus.trigger("initials_change", state.initialsExtra);
```
Comment (joamag) [inline vue/components/personalization.vue]: why this change? we should have proper initials in the event right away.

Comment (joamag) [reply vue/components/personalization.vue]: Yup, I've just analyzed this, there's a huge difference in semantics. This is not a proper fix.

File: vue/components/personalization.vue (hunk 1 above)
Comment (joamag) [reply vue/components/personalization.vue]: Yup, I've just analyzed this, there's a huge difference in semantics. This is not a proper fix.

Comment (gcandal) [reply vue/components/personalization.vue]: Initials changed = the UI initials changed
Initials extra = the SDK initials changed

In here we want to subscribe to changes in the UI definition, to keep the several instances in sync.

File: vue/components/personalization.vue (hunk 2 above)
Comment (gcandal) [reply vue/components/personalization.vue]: Because of the comment below. If this is the event for a UI-level change then the sanitisation shouldn't happen here
